### DISCUSSION TITLE
Turn off warning about outdated version of Pluto

### DISCUSSION THREAD

**Discussion by KronosTheLate** [fonsp/Pluto.jl#943] [Questions about using Pluto] [answered]
I am writing a pluto script that is supposed to be a steamlined experience for an end-user that does not know any programming, and want things to be kept as simple as possible.

In that spirit, I would like it is there was not a warning to update pluto upon launch - I am creating a sysimage, and want the setup to be plug-and-play for as long as possible. Is it currently possible to disable the warning? If not, is it possible to add a flag for it?

**Comment by fonsp** [comment, marked as answer]
No flag yet, but feel free to open a PR!
